Simplified/NYPLSettingsAccountDetailViewController.m

```diff
@@ -578,6 +578,7 @@ - (void)validateCredentials
   [task resume];
 }
 
+#if defined(FEATURE_DRM_CONNECTOR)
```

Comment:
this also seems like it should stay?

Reply:
Function in this block is used only for Adobe builds and doesn't compile otherwise. From history of the file I see, that this method was refactored from other larger method, and it was embedded in #if check back then.

Reply:
I see, makes sense.